source/agora/node/BlockStorage.d

```diff
@@ -812,6 +812,9 @@ public class BlockStorage : IBlockStorage
 
     private static ubyte[] makeChecksum (const ubyte[] data) @safe nothrow
     {
+        assert(data.length < 1 << 20,
```

Comment:
` assert((MapSize <= 1 << 20) && ((data.length + ChecksumSize) <= 1 << 20),`
We can make file up to 1MB.
If the map size is changed, I think we can catch the error.

Reply:
This isn't really testing for the size of the map, it's testing the size of the data that is being checksummed. That's why this function is `static`, it doesn't care about what the "map" size is. The reason for this limit is because CRC32 has a limited efficiency based on the size of its input data.

There is more information here: https://stackoverflow.com/questions/2321676/data-length-vs-crc-length

Reply:
As you say, mapsize doesn't seem right to do check here.
However, I wish the data size + checksum size would be 1MB.
I think it will be good to expand to 1MB when changing the mapsize in the future. 
It is convenient to change the mapsize to `size_t MapSize = 1024 * 1024;`

Reply:
I see, in that case I'll add a check for that.